index.bs

```diff
@@ -4558,6 +4558,23 @@ FIDO APIs use an alternative identifier for [=[RPS]=] called an |AppID|
 that identifier. Without this extension, they would need to be re-registered in
 order to be [=scoped=] to an [=RP ID=].
 
+In addition to setting the {{AuthenticationExtensionsClientInputs/appid}} extension input,
+using this extension requires some additional processing by the [=[RP]=]
+in order to [=authentication|authenticate=] U2F credentials:
+
+ 1. List the desired U2F credentials in the {{PublicKeyCredentialRequestOptions/allowCredentials}} option
+     of the {{CredentialsContainer/get()}} method:
+
+     - Set the {{PublicKeyCredentialDescriptor/type}} members to {{PublicKeyCredentialType/public-key}}.
+     - Set the {{PublicKeyCredentialDescriptor/id}} members to the respective U2F key handles of the desired credentials.
+
+     {{PublicKeyCredentialRequestOptions/allowCredentials}} MAY contain a mixture
+     of both WebAuthn [=credential IDs=] and U2F key handles;
+     setting the {{AuthenticationExtensionsClientInputs/appid}} does not prevent the user from using a WebAuthn credential.
+
+ 1. When [verifying the assertion](#sctn-verifying-assertion), expect that the <code>[=rpIdHash=]</code>
+     MAY be the hash of the |AppID| instead of the [=RP ID=].
```

Comment:
I wonder whether we ought to add a Note: referencing this extension to #sctn-verifying-assertion on the appropriate step(s)?

Reply:
There's a link to `#sctn-verifying-assertion` right there. :) But it's a good idea to link directly to the relevant step, so I added an anchor to step 6 and updated the links to point directly there. Seem good?

Reply:
sorry, I wasn't clear: what I meant was to add a Note to that 'rp-op-verifying-assertion-step-rpid-hash' step in #sctn-verifying-assertion that points back over here and says something like "if you use the AppId extension, see here for additionall processing details..."

Reply:
> so I added an anchor to step 6 and updated the links to point directly there

nice :)

Reply:
I added a note to the RP assertion verification steps, does that look good?